.gitignore

```diff
@@ -28,5 +28,6 @@ serverless_config.json
 yalc.lock
 
 /implementationGuides
-/compiledImplementationGuides
+/compiledImplementationGuides/*
+!/compiledImplementationGuides/gitkeep
```

Comment:
Nitpick: Should this be `.gitkeep`. That's how I normally see the file added. I think `gitkeep` would work too.

Benefit of `.gitkeep` is it wouldn't normally show up when opening that directory unless someone decides to show hidden files. That reduces the chance of the someone deleting the file.

Reply:
.gitkeep won’t work in this particular case. It would keep the folder in  git but the copy files glob would still fail since glob does not match hidden files